test-scripts/test.ts

```diff
@@ -23,6 +23,7 @@ import { env } from 'process'
             vscodeExecutablePath: vsCodeExecutablePath,
             extensionDevelopmentPath: cwd,
             extensionTestsPath: testEntrypoint,
+            launchArgs: ['--verbose', '--log', 'debug'],
```

Comment:
I sugggest we leave this in here (commit it to master) for awhile, it doesn't seem to bloat the logs too much and may be useful as we continue to eliminate CI unreliability.

Reply:
Sounds good. Please add a comment above this line stating this intent, and consider providing an in-comment ETA (2020-12 ?) of when to revisit + corresponding backlog task.